networks/remote/README.md

```diff
@@ -0,0 +1 @@
+# Remote Cluster with Terraform and Ansible
```

Comment:
see docs/terraform-and-ansible.rst

Reply:
ok fixed in https://github.com/tendermint/tendermint/pull/1602 thanks